components/esm/aipackage.cpp

```diff
@@ -7,8 +7,7 @@ namespace ESM
 {
     void AIData::blank()
     {
-        mHello = mU1 = mFight = mFlee = mAlarm = mU2 = mU3 = mU4 = 0;
-        mServices = 0;
+        *this = AIData();
```

Comment:
Are you sure that this will work correctly? AFAIK value-initialization of structs is a C++11 only feature. Better set the values manually for now.